### ISSUE TITLE
Wiki gone?

### ISSUE THREAD

**Issue by gdsl7** [chkr1011/CoAPnet#13]
I can't find the wiki with examples any more. Is it gone ?

**Comment by chkr1011** [comment]
Yes the Wiki is one. I will add a samples project containing several samples. Please check the file https://github.com/chkr1011/CoAPnet/blob/master/Source/CoAP.TestClient/Program.cs as long as the samples are not yet available.

**Comment by gdsl7** [comment]
Thank you.

Any chance the sample could be updated to compile with the latest build? 

The DTLS changes breaks the sample app.

**Comment by chkr1011** [comment]
Please share the build error you get. I can compile without issues.
